modules/rosa-rn-platform.adoc

```diff
@@ -0,0 +1,70 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-platform_{context}"]
+= Platform
+
+[id="rosa-rn-backups_{context}"]
+== Cluster backups
+
+Application and application data backups are not a part of the {product-title} (ROSA) service. Customers must have their own backup plan for their applications and application data.
+
+All Kubernetes objects and persistent volumes (PVs) in each ROSA cluster are backed up to facilitate a prompt recovery in the unlikely event that a cluster becomes irreparably inoperable. These backups are stored in a secure object storage (multiple availability zone) bucket in the same account as the cluster.
+
+Node root volumes are not backed up because Red Hat Enterprise Linux CoreOS is fully managed by the OpenShift Container Platform cluster and no stateful data should be stored on a node’s root volume.
+
+[id="rosa-rn-autoscaling_{context}"]
+== Autoscaling
+
+Autoscaling is available through the `rosa` command-line. ROSA supports autoscaling on nodes. Autoscaling can be configured during cluster creation or on machine pools after a cluster is created.
+
+[id="rosa-rn-daemonsets_{context}"]
+== Daemonsets
+
+Customers can create and run daemonsets on ROSA. Daemonsets can be limited to run on worker nodes using `nodeSelector`.
+
+[id="rosa-rn-multi-az_{context}"]
+== Multiple availability zone
+
+In a multiple availability zone cluster, master nodes are distributed across availability zones and one worker node is required in each availability zone.
+
+[id="rosa-rn-node-labels_{context}"]
+== Node labels
+
+Custom node labels are created by Red Hat during node creation and cannot be changed on ROSA clusters at this time.
+
+[id="rosa-rn-versions_{context}"]
+== OpenShift Container Platform Versions
+
+ROSA runs as a service on Amazon Web Services (AWS) and is kept up-to-date with the latest OpenShift Container Platform version.
+
+[id="rosa-rn-upgrades_{context}"]
+== Upgrades
```

Comment:
Please direct to the Life Cycle page: https://access.redhat.com/support/policy/updates/openshift/dedicated

Reply:
Done.